altizon/io/datonis/edge/edge_gateway_mqtt.py

```diff
@@ -45,8 +66,7 @@ def instruction_worker(thread_name,gateway):
         try:
             instruction_dispatcher(gateway) 
         except:
-            e = sys.exc_info()[0]
-            logging.error('instruction_dispatcher failed', str(e))
+            logging.error('instruction_dispatcher failed', exc_info=True)
```

Comment:
Did not understand what happens here. Does it log the full exception?

Reply:
Yes, logging module print complete stack trace if exc_info parameter is passed.